### ISSUE TITLE
Error getTensorContentRef_ when inference UNet Keypoints model Keras from cv2.dnn.readNetFromTensorflow

### ISSUE THREAD

**Issue by slavazhulanov** [opencv/opencv#23500]
### System Information

OpenCV python version: 4.7.0
Operating System / Platform: Win 10
Python version: 3.11.0

### Detailed description

error: OpenCV(4.7.0) D:\a\opencv-python\opencv-python\opencv\modules\dnn\src\tensorflow\tf_graph_simplifier.cpp:901: error: (-215:Assertion failed) !field.empty() in function 'cv::dnn::dnn4_v20221220::getTensorContentRef_'

### Steps to reproduce

1. trained UNet to detect keypoints in Keras and saved in SavedModel (.pb) format
![image](https://user-images.githubusercontent.com/50666828/232470424-aba40703-2215-4311-bd23-cb0edb7dbd2c.png)
2. created a frozen graph 
![image](https://user-images.githubusercontent.com/50666828/232470596-208b84f8-08cc-43dd-99bb-ad95ce76d4e7.png)
3. Run neural network in OpenCV using dnn.ReadNetFromTensorflow module
![image](https://user-images.githubusercontent.com/50666828/232470273-61e22eba-206a-44ce-8f9f-ded346558f04.png)

### Issue submission checklist

- [X] I report the issue, it's not a question
- [X] I checked the problem with documentation, FAQ, open issues, forum.opencv.org, Stack Overflow, etc and have not found any solution
- [X] I updated to the latest OpenCV version and the issue is still there
- [X] There is reproducer code and related data files (videos, images, onnx, etc)

**Comment by slavazhulanov** [comment]
My [model](https://drive.google.com/file/d/1QIZZDnd2uEcbmOkoJ8AO0FOS7iAO7QfO/view?usp=sharing)

**Comment by WanliZhong** [comment]
@slavazhulanov I found your model have a problem. The begin of `stridedSlice` layer is empty. So I think you should look into how the model was generated.
![image](https://user-images.githubusercontent.com/49567307/233033829-7912f3d4-f0f6-47a4-a181-b82aff733906.png)

**Comment by slavazhulanov** [comment]
> So is the issue solved or still has some bugs?

I would like to know which layers Keras supports in OpenCV? perhaps I should replace some of the layers in my model?

**Comment by WanliZhong** [comment]
> > So is the issue solved or still has some bugs?
> 
> I would like to know which layers Keras supports in OpenCV? perhaps I should replace some of the layers in my model?

I am not sure if there are some doc. You can go to [tf_importer.cpp](https://github.com/opencv/opencv/blob/4.x/modules/dnn/src/tensorflow/tf_importer.cpp) and search inside this file to look if the operator is supported.

**Comment by slavazhulanov** [comment]
Hi @WanliZhong 
I would like to clarify what the label assigned to this issue means? will my problem be solved in the next version of OpenCV?
![image](https://user-images.githubusercontent.com/50666828/233618805-26d949f1-6388-48a1-8570-601bfc2f4083.png)

**Comment by WanliZhong** [comment]
@slavazhulanov Actually, I am not sure what is your problem? Your model is not correct because the data is empty. So I can fix nothing. If there has something worng about OpenCV?

**Comment by slavazhulanov** [comment]
@WanliZhong 
it's strange, when I load the model into Keras it works. Providing access to the [model](https://drive.google.com/drive/folders/1LTnkGQydAP0zUei8TMV64ycSH9RAEiqo?usp=sharing) saved in SavedModel.pb format

**Comment by fengyuentau** [comment]
I tested your onnx model in https://github.com/opencv/opencv/issues/23500#issuecomment-1515914897, it works.

In your latest comment https://github.com/opencv/opencv/issues/23500#issuecomment-1519775113, the model is in a fancy format but not the freeze weight format. Could you provide the model in freeze weight format?

**Comment by slavazhulanov** [comment]
providing a link to a [frozen model graph](https://drive.google.com/file/d/1g_znSnmtYfANfM9Ua-4VgM5tOXXw52Jl/view?usp=sharing)

**Comment by asmorkalov** [comment]
@fengyuentau @WanliZhong Could you take a look on the last model version and provide comment. Looks like OpenCV patches are not expected here.

**Comment by fengyuentau** [comment]
There is not much documentation for `strided_slice` operator from TensorFlow. The problem is as described in https://github.com/opencv/opencv/issues/23500#issuecomment-1514430004. No patches should be made in this case.

**Comment by asmorkalov** [comment]
Ok, close the issue then.
